ui/admin/tests/acceptance/credential-store/credentials/update-test.js

```diff
@@ -42,58 +46,150 @@ module(
         scope: instances.scopes.project,
         type: 'static',
       });
-      instances.credential = this.server.create('credential', {
+      instances.usernamePasswordCredential = this.server.create('credential', {
         scope: instances.scopes.project,
         credentialStore: instances.staticCredentialStore,
         type: 'username_password',
       });
+      instances.usernameKeyPairCredential = this.server.create('credential', {
+        scope: instances.scopes.project,
+        credentialStore: instances.staticCredentialStore,
+        type: 'ssh_private_key',
+      });
       // Generate route URLs for resources
       urls.projectScope = `/scopes/${instances.scopes.project.id}`;
       urls.credentialStores = `${urls.projectScope}/credential-stores`;
       urls.staticCredentialStore = `${urls.credentialStores}/${instances.staticCredentialStore.id}`;
       urls.credentials = `${urls.staticCredentialStore}/credentials`;
-      urls.credential = `${urls.credentials}/${instances.credential.id}`;
+      urls.usernamePasswordCredential = `${urls.credentials}/${instances.usernamePasswordCredential.id}`;
+      urls.usernameKeyPairCredential = `${urls.credentials}/${instances.usernameKeyPairCredential.id}`;
       authenticateSession({});
     });
 
-    test('can save changes to existing credential', async function (assert) {
+    test('can save changes to existing username & password credential', async function (assert) {
       assert.expect(3);
-      assert.notEqual(instances.credential.name, 'random string');
-      await visit(urls.credential);
+      assert.notEqual(
+        instances.usernamePasswordCredential.name,
+        'random string'
```

Comment:
It would be nice if this `random string` was pulled out to a local variable so it's the same random string that gets tested

Reply:
Good Idea! I will do that for other acceptance test files as well.